pilot/pkg/serviceregistry/cloudfoundry/serviceaccounts.go

```diff
@@ -14,10 +14,11 @@
 
 package cloudfoundry
 
-import "istio.io/istio/pilot/pkg/model"
```

Comment:
nit: this could be kept as is.

Reply:
cloud i resolve this ?

Reply:
Yes, I generally refer the commit by ID when resolving comments.